guides/private-app/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "carto-app-template",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite --https",
```

Comment:
I would prefer `npm run start` to be consistent with deck.gl

Reply:
`npm run dev` is kind of the default with vite, but up to you guys, no strong pref

Reply:
I prefer compatibility with vite. This repo is the default of a guide. 

https://app.gitbook.com/o/zP3LT2Qhlb5jPWQCfWV3/s/ybPdpmLltPkzGFvz7m8A/~/changes/393/carto-for-developers/guides/build-a-public-application

And I don't want to make the guide larger just to modify a command.

Reply:
My bad, because I didn't add enough context to the PR description